docs/changelog/changelog.rst

```diff
@@ -26,6 +26,9 @@ New Features
   :meth:`~holodeck.environments.HolodeckEnvironment.reset`. See
   :ref:`location-randomization`.
   (`#295 <https://github.com/BYU-PCCL/holodeck/issues/295>`_)
+- :meth:`~holodeck.environment.spawn_prop` now allows for programmatically
+  spawning basic Unreal actors into the environment.
```

Comment:
I don't know if this is the best description. Will whoever reads this understand what we mean by "Unreal actors"? Maybe

>:meth:`~holodeck.environment.spawn_prop` now allows basic objects (spheres, cubes, cylinders)
>  to be spawned at arbitrary locations in the environment